### DISCUSSION TITLE
Object Storage

### DISCUSSION THREAD

**Discussion by snadrus** [filecoin-project/curio#223] [Ideas]
MinIO provides solutions for large scale stability, but this has no compatibility with Curio yet. 
This is needed in long-term storage, so the Curio Batch Sealing system won't include object storage until the push to long-term.

**Comment by beck-8** [comment]
Just a reminder that although minio has a good reputation, it has a lot of problems, including large-scale performance issues, and the official does nothing. Issues raised are immediately closed (you need to pay them).

**Reply by LexLuthr** [reply]
MinIO is just one implementation. Curio will probably do a generic implementation to allow plugging-in any object storage which adheres to certain standards.
